Source/WebCore/platform/mediastream/cocoa/DisplayCaptureSourceCocoa.cpp

```diff
@@ -141,7 +141,7 @@ const RealtimeMediaSourceSettings& DisplayCaptureSourceCocoa::settings()
         auto size = this->size();
         settings.setWidth(size.width());
         settings.setHeight(size.height());
-        settings.setDeviceId(hashedId());
+        settings.setDeviceId(AtomString { hashedId() });
```

Comment:
Should we change `RealtimeMediaSource::hashedId()` to be an `AtomString` instead?

Reply:
Just checked and yes. I'll fix.